component-samples/demo/owner/serviceinfo/sample-svi.csv

```diff
@@ -1 +1 @@
-sdo_sys:filedesc=payload_name,sdo_sys:write=payload.bin,sdo_sys:filedesc=package_name,sdo_sys:write=package.sh,sdo_sys:exec=binsh-linux64,sdo_wget:filename=sample_file,sdo_wget:url=url
\ No newline at end of file
+fdo_sys:filedesc=payload_name,fdo_sys:write=payload.bin,fdo_sys:filedesc=package_name,fdo_sys:write=package.sh,fdo_sys:exec=binsh-linux64,fdo_wget:filename=sample_file,fdo_wget:url=url
```

Comment:
This file can be deleted now, since we're no longer using it. We'd also need to document this mapping information in Owner's README.